tests/run-pass/time.rs

```diff
@@ -4,15 +4,19 @@
 use std::time::{SystemTime, Instant};
 
 fn main() {
+
```

Comment:
What's with the newline here?
Also you need to remove `ignore-windows` if this is supposed to test anything. :)